package.json

```diff
@@ -54,6 +54,7 @@
     "rollup": "^1.27.9",
     "rollup-plugin-babel": "^4.3.3",
     "rollup-plugin-copy": "^3.1.0",
+    "rollup-plugin-extensions": "^0.1.0",
```

Comment:
Fixes issue related to https://github.com/Microsoft/TypeScript/issues/27481 and others.

Basically, TS won't let you use extensions but rollup doesn't look for `.ts`/`.tsx` files out of the box, so in the first commits I just threw `@ts-ignore` on to get rollup to build - but that also hid some minor TS type import/export issues